amt.go

```diff
@@ -495,11 +495,14 @@ func (n *Node) Flush(ctx context.Context, bs cbor.IpldStore, depth int) error {
 		if len(n.expVals) == 0 {
 			return nil
 		}
-		n.Bmap = []byte{0}
+		n.Bmap = nil
```

Comment:
always defaulting to having the bitfield set would just mean moving this line (before this change) to above the `if`